modules/telco-upgrade-monitoring-second-part-y-upgrade.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-monitoring-second-part-y-upgrade_{context}"]
+= Monitoring the second part of a <y+1> cluster upgrade
+
+Follow the same steps as described in "Monitoring the cluster upgrade" except this time track the update for the second part of the <y+1> upgrade.
```

Comment:
It seems like you're giving most of the same information here anyways, so why mention this? Or if you are actually expecting them to repeat something that would make them have to go backwards, then I think you should just repeat it here, so they don't have to jump back and forth.

(And if there really is something different in this module, but it's mostly the same, you could always share the same module, but just with replaceable values, or perhaps ifdefs).